### ISSUE TITLE
Environment variables with ${...} in them cannot be used in a manifest file

### ISSUE THREAD

**Issue by davidehringer** [cloudfoundry/cli#682]
In a manifest file, when you include an environment variable with ${...} in it, the manifest fails validation. For example:

```

---
applications:
-   name: example-env
    env:
        EXAMPLE: xyx${something}abc
```

results in `Error reading manifest file:
Property '${something}' found in manifest. This feature is no longer supported. Please remove it and try again.`

My usage isn't intended for property expansion but rather to passed through without modification (e.g. characters in a password). Is the desired behavior to not allow ${} anywhere in the manifest file (excluding ${random-word})?

**Comment by krishicks** [comment]
Hi @davidehringer,

This is the expected behavior presently. This was done in 4ba4a9f, which dates to almost two years ago.

@scottruitt might have some more background, if not @tedsuo. It would be up to @dkoper to decide if we want to bring this back, and how.

**Comment by dkoper** [comment]
Hi @davidehringer,

I believe the background was to discourage people from relying on such env vars to be resolved with information from the container, which may change over time and differ between staging and running containers.

Regards,
Dies Koper

**Comment by schulzh** [comment]
We just ran into the same issue. Our manifest looks like this;
```
---
applications:
- name: grafana
  [...]
  command: |
    sed -i "/\[database\]/a url = \${DATABASE_URL}" /etc/grafana/grafana.ini
    /run.sh
```
The command should put the literal line `url = ${DATABASE_URL}` in the grafana configuration, so grafana takes the url from the environment at runtime, but the cli does not push the application because it finds a "property" which it wont expand.

There needs to be some kind of option to tell the CF cli to still push the application.

**Comment by dkoper** [comment]
I'd like to look at how we can let such values just pass through.

**Comment by dkoper** [comment]
@davidehringer @schulzh This should work now with `cf v2-push` on cf CLI 6.33.0. Please try.

**Comment by dkoper** [comment]
This is now resolved in cf CLI 6.34.0.

Regards,
Dies Koper
CF CLI PM
